test/Tests/Core/SpecialFunctionsTest.cs

```diff
@@ -23948,6 +24000,18 @@ private static void CheckFunctionValues(string name, Delegate fcn, double[,] pai
                     x[k] = pairs[i, k];
                     args[k] = x[k];
                 }
+                bool showTiming = false;
```

Comment:
Was it the intention to have this performance measurement logic in master? If yes, it needs to be moved into a separate method and used in tests.

Reply:
It is in tests.